packages/Manager/.batect/database/init.sql

```diff
@@ -1,8 +1,18 @@
 CREATE DATABASE eop_dev;
 CREATE DATABASE eop_test;
 
+
+-- Setup dev with the 2 users we use for running the App vs Migrations
+-- these will be manually setup in the deployment environments
 \c eop_dev
 CREATE EXTENSION postgis;
 
+REVOKE ALL PRIVILEGES ON SCHEMA public FROM PUBLIC;
+
+CREATE USER eop_manager_migrations_user WITH password 'password' NOINHERIT;
+GRANT ALL ON SCHEMA public TO eop_manager_migrations_user WITH GRANT OPTION;
+
+CREATE USER eop_manager_app_user WITH password 'password' NOINHERIT;
+
```

Comment:
This section needs to be run manually against a new RDS instance

Reply:
Is it worth noting this somewhere else too? Not sure if we have a setup guide, or perhaps just a comment inline for now?

Reply:
Done,
Main thing was adding a section to the README about boot-strapping a new server ... but I've also added similar comments to this init.sql